sw/device/tests/dif/dif_plic_sanitytest.c

```diff
@@ -45,37 +38,32 @@ static volatile bool uart_handled_more_than_once;
  * determine success or failure of the test.
  */
 static void handle_uart_isr(const dif_plic_irq_id_t interrupt_id) {
-  const dif_uart_t *uart = &uart0;
+  // NOTE: This initialization is superfluous, since the `default` case below
+  // is effectively noreturn, but the compiler is unable to prove this.
+  dif_uart_interrupt_t uart_irq = 0;
```

Comment:
No, but it started complaining after I did a refactor. Before, the compiler was able to prove that the assignment happened, because you had a superfluous abort() after set_test_status(). abort is noreturn, set_test_status() is not.

Reply:
Makes sense.